geometry_msgs/Twist velocity
float64 distance_traveled
geometry_msgs/Pose position
float32 battery_level
builtin_interfaces/Duration travel_time

